### DISCUSSION TITLE
Transferring IOTA from Trinity wallet to Firefly wallet

### DISCUSSION THREAD

**Discussion by Bogdan4505** [iotaledger/firefly#7745] [Help & Support]
Hello
I have my IOTA stored on my Trinity wallet. I did not open this wallet for while until today, when I realized this wallet was discontinued. I'm not sure if there is a way to transfer my IOTA to new Firefly wallet. I have my seed and kdbx file. Please advise 
Thanks

**Comment by begonaalvarezd** [comment]
The current ability to auto-migrate tokens from the pre-Chrysalis IOTA network via the Firefly wallet have been stopped with the IOTA Stardust upgrade. Once IOTA smart contracts are available on the IOTA network, this feature will be restored and migrations can be processed automatically again
We will provide a migration tool to resume legacy migrations, but its not finished yet

Will close this issue because Firefly will not offer legacy migration but it will be a different tool, we will announce it when the tool is ready

**Reply by Bogdan4505** [reply]
Thanks

**Comment by Bogdan4505** [comment]
ok, thanks
________________________________
